### PR TITLE
newnew

### PR DESCRIPTION
**Pull request by ghost**
newnewnew

### CODE REVIEW

**Comment by ghost** [comment]
יגבר

**Comment by ghost** [comment]
יגגבבררררר

File: index.php [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <title>Document</title>
 </head>
 <body>
-    
+    <p>newnewnewnew</p>
     <?php
         include './serve/navbar.php';
```
Comment (jayber-dev) [inline index.php:14]: what is that madafucka

Comment (jayber-dev) [reply index.php:14]: you mama paka paka

**Review by jayber-dev — APPROVED** [review]

**Pull request closed without merging**